core/src/main/java/com/flipkart/gjex/core/web/RotationManagementResource.java

```diff
@@ -0,0 +1,52 @@
+package com.flipkart.gjex.core.web;
+
+import com.flipkart.gjex.core.task.RotationManagementBasedHealthCheck;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.servlet.ServletContext;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+/**
+ * Resource for rotation management
+ * @author ajay.jalgaonkar
+ */
+
+@Singleton
+@Path("/")
+@Named
+public class RotationManagementResource {
+
+  @Context
+  private ServletContext servletContext;
```

Comment:
This is not used . Can we please remove it

Reply:
removed